ql/src/java/org/apache/hadoop/hive/ql/parse/StorageFormat.java

```diff
@@ -245,4 +237,16 @@ public Map<String, String> getSerdeProps() {
   public void setStorageHandler(String storageHandlerClass) throws SemanticException {
     storageHandler = ensureClassExists(storageHandlerClass);
   }
+
+  public static StorageFormatDescriptor getStorageFormatDescriptor(String format, String clause)
```

Comment:
maybe simply `getDescriptor` as it's defined inside of StorageFormat class>

Reply:
done